maps/kleibkhar/kleibkhar_exoplanet.dm

```diff
@@ -0,0 +1,108 @@
+#define EXT_EDGE_GRASS_KLEIBKHAR    (53 * 0.001)
```

Comment:
Shouldn't this be EXT_LAYER_CONSTANT or something?

Reply:
This follows from the other layer defines, but its more explicit in those files I guess. I'll go ahead and change it.

Reply:
EXT_LAYER_CONSTANT is undefined in an earlier file, so I can't use it here.